apps/prairielearn/elements/pl-threejs/pl-threejs.py

```diff
@@ -494,17 +494,17 @@ def grade(element_html, data):
 def parse_correct_answer(f, a):
     if f == "homogeneous":
         try:
-            T = np.array(a, dtype=np.float64)
-            if T.shape == (4, 4):
-                R = T[0:3, 0:3]
-                p = T[0:3, 3:4]
-                return np.reshape(p, (3,)), pyquaternion.Quaternion(matrix=R)
+            transform_mat = np.array(a, dtype=np.float64)
```

Comment:
Elsewhere we use `matrix` as a variable name; should we use `transform_matrix`/`rotate_matrix` instead here?

Reply:
Resolved in cb0dd63df